model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -213,17 +213,40 @@ message JobMessagesResponse {
 // without needing to pass through STARTING.
 message JobState {
   enum Enum {
+    // The job state reported by a runner cannot be interpreted by the SDK.
     UNSPECIFIED = 0;
+
+    // The job has been paused, or has not yet started.
```

Comment:
Didn't we decide that there is no pause/restart? 

Stopped really sounds like the job was running at one time, until someone decided to stop it. Perhaps this would be worth renaming to UNSTARTED?

Reply:
I removed "paused" from the description.  I agree that the name STOPPED is misleading, but renaming it would be a much bigger task that could leak into non-portable runners.  Hopefully the improvement surrounding the documentation of these enums will suffice for now.